### PR TITLE
Subscriber importing: split to 3 questions

### PR DESCRIPTION
**Pull request by simison**
Resolves https://github.com/Automattic/jetpack-roadmap/issues/1896

Internal reference to designs peKye1-17s-p2

**Before**

image

**After**

Screenshot 2024-11-19 at 11 15 03
Screenshot 2024-11-19 at 11 15 10
Screenshot 2024-11-19 at 11 57 56

Before Subscriber importer is released, it stays hidden:

Screenshot 2024-11-19 at 12 18 07

No changes to onboarding flow screen at `/setup/newsletters`:

Screenshot 2024-11-19 at 11 36 13

## Proposed Changes

* Splits subscriber importing into three distinct ways:
  - Add one by one as emails
  - Upload CSV
  - Import from Substack

## Why are these changes being made?

* Creates a clear entrypoint to guided paid subscriber importing from Substack, separately from free subscriber importing at this screen.

## Testing Instructions

*

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: packages/subscriber/src/components/flow-card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { Badge } from '@automattic/components';
+import { Flex, FlexBlock, FlexItem, Card, CardBody, Icon } from '@wordpress/components';
+import { chevronRight } from '@wordpress/icons';
+import clsx from 'clsx';
+import type { BadgeType } from '@automattic/components';
+import './style.scss';
+
+interface FlowCardProps {
```
Comment (simison) [inline packages/subscriber/src/components/flow-card/index.tsx:8]: Copied from Stepper components; we can move to `@automattic/components` likely.

**Comment by crisbusquets** [comment]
I see it's still a draft, but the hover state here should use the `Blueberry`, or the accent color they have:

container-funny-volhard calypso live_subscribers_songokunewsletter wordpress com

**Comment by simison** [comment]
Two follow-ups for after merging:

- More typography size adjustments
- https://github.com/Automattic/wp-calypso/issues/96524
- https://github.com/Automattic/wp-calypso/pull/96235

**Comment by simison** [comment]
> the hover state here should use the Blueberry, or the accent color they have:

Which one is the Blueberry blue from [color studio](https://color-studio.blog/)? I'm using `--studio-blue-50` currently.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/17008523

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @simison for including a screenshot in the description! This is really helpful for our translators.

**Comment by lezama** [comment]
modal is looking good! I am getting this when skipping both:

Screenshot 2024-11-19 at 8 33 09 AM

**Review by lezama — APPROVED** [review]
nevermind, works well in prod.

:shipit:

File: packages/subscriber/src/components/flow-card/index.tsx (hunk 1 above)
Comment (simison) [reply packages/subscriber/src/components/flow-card/index.tsx:8]: PR: https://github.com/Automattic/wp-calypso/pull/96235

**Pull request merged**